src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java

```diff
@@ -145,11 +145,11 @@ public void onPlayerChat(PlayerChatEvent event) {
       }
 
       var message = new StringTemplate(config.discord.MESSAGE_FORMAT.get())
-        .add("username", username)
-        .add("server", server)
-        .add("message", content).toString();
+              .add("username", username)
+              .add("server", server)
+              .add("message", content).toString();
 
-      sendMessage(message);
+      webhookClient.sendMessage(message);
```

Comment:
I'm unsure what this change exactly does. Can you elaborate?

Reply:
> I'm unsure what this change exactly does. Can you elaborate?

I'm not a developer and don't understand anything about code. Idea intellij showed a bug and suggested me to do this to fix it.